README.md

```diff
@@ -226,7 +226,7 @@ PostPresenter::make($post)->preset('summary')->with(function () {
 });
 ```
 
-#### Caveat
+**Caveat: repeat call method**
```

Comment:
Could we change this to:

Caveat: repeated method calls

Reply:
OK